packages/maskbook/src/utils/hooks/useBase64.ts

```diff
@@ -0,0 +1,17 @@
+import { useAsyncRetry } from 'react-use'
+import Services from '../../extension/service'
+
+export function useBase64(url: string) {
+    return useAsyncRetry(async () => {
```

Comment:
I think we can add a cache here to avoid calculate the same image time and time again.

Reply:
Ummm... Sounds great!

Reply:
I have a question. Why you need to turn the image into base64?